operator/pkg/central/extensions/reconcile_admin_password_test.go

```diff
@@ -103,6 +106,12 @@ func TestReconcileAdminPassword(t *testing.T) {
 					assert.True(t, hf.Check(basic.DefaultUsername, "foobarbaz"))
 				},
 			},
+			VerifyStatus: func(t *testing.T, status *platform.CentralStatus) {
+				require.NotNil(t, status.Central)
+				require.NotNil(t, status.Central.AdminPassword)
+				assert.Contains(t, status.Central.AdminPassword.Info, "The admin password is configured to match")
+				assert.Contains(t, status.Central.AdminPassword.SecretReference, "my-password")
```

Comment:
I've been starting at this PR for the last 5 minutes and I cannot figure out how this test case can possibly be passing?
I can see the only place where `.SecretReference` is being is set is from `r.passwordSecretName` [here](https://github.com/stackrox/stackrox/pull/1206/files#diff-0fdb3bf9ba7d7186b84a925524c29d41640249c259061d3a7dc9f6e262c631a1R122) and that field in turn is only ever set to the `defaultPasswordSecretName` constant on lines 152, 155 and 182.
Can you please explain that to me?